chapters/es/chapter3/2.mdx

```diff
@@ -0,0 +1,384 @@
+
+Este comando descarga y memoriza el conjunto de datos, por defecto en *~/.cache/huggingface/dataset*. Recuerda del Capítulo 2 que puedes personalizar tu carpeta mediante la configuración de la variable de ambiente `HF_HOME`.
+
+Podemos acceder cada par de oraciones en nuestro objeto `raw_datasets` usando indexación, como con un diccionario.
+
+```py
+raw_train_dataset = raw_datasets["train"]
+raw_train_dataset[0]
+```
+
+```python out
+{'idx': 0,
+ 'label': 1,
+ 'sentence1': 'Amrozi accused his brother , whom he called " the witness " , of deliberately distorting his evidence .',
+ 'sentence2': 'Referring to him as only " the witness " , Amrozi accused his brother of deliberately distorting his evidence .'}
+```
+
+Podemos ver que las etiquetas ya son números enteros, así que no es necesario hacer ningún preprocesamiento. Para saber cual valor corresponde con cual etiqueta, podemos inspeccionar el campo `features` de nuestro `raw_train_dataset`. Esto indicara el tipo de cada column:
+
+```py
+raw_train_dataset.features
+```
+
+```python out
+{'sentence1': Value(dtype='string', id=None),
+ 'sentence2': Value(dtype='string', id=None),
+ 'label': ClassLabel(num_classes=2, names=['not_equivalent', 'equivalent'], names_file=None, id=None),
+ 'idx': Value(dtype='int32', id=None)}
+```
+
+Internamente, `label` es de tipo `ClassLabel`, y la asociación de valores enteros y sus etiquetas esta almacenado en la carpeta *names*. `0` corresponde con `not_equivalent`, y `1` corresponde con `equivalent`.
+
+<Tip>
+
+✏️ **Inténtalo!** Mira el elemento 15 del conjunto de datos de entrenamiento y el elemento 87 del conjunto de datos de validación. Cuáles son sus etiquetas?
+
+</Tip>
+
+### Preprocesando un conjunto de datos
+
+{#if fw === 'pt'}
+<Youtube id="0u3ioSwev3s"/>
+{:else}
+<Youtube id="P-rZWqcB6CE"/>
+{/if}
+
+Para preprocesar el conjunto de datos, necesitamos convertir el texto en números que puedan ser entendidos por el modelo. Como viste en el [capítulo anterior](/course/chapter2), esto se hace con el toquenizador. Podemos darle al tokenizador una oración o una lista de oraciones, así podemos tokenizar directamente todas las primeras oraciones y todas segundas oraciones de cada par de la siquiente manera:
+
+```py
+from transformers import AutoTokenizer
+
+checkpoint = "bert-base-uncased"
+tokenizer = AutoTokenizer.from_pretrained(checkpoint)
+tokenized_sentences_1 = tokenizer(raw_datasets["train"]["sentence1"])
+tokenized_sentences_2 = tokenizer(raw_datasets["train"]["sentence2"])
+```
+
+Sin embargo, no podemos simplemente pasar dos secuencias al modelo y obtener una predicción indicando si estas son parafrases o no. Necesitamos manipular las dos secuencias como un par y aplicar el preprocesamiento apropiado.
+Afortunadamente, el tonenizador también puede tomar un par de secuencias y preparalo de la manera como nuestro modelo BERT lo espera:
```

Comment:
I wrote: Afortunadamente, el **tokenizador** puede recibir también un par de oraciones y **preparar** las misma de una forma que nuestro modelo BERT espera.

Reply:
Like it.